pkg/shell/hosts.jsx

```diff
@@ -119,6 +119,8 @@ export class CockpitHosts extends React.Component {
             this.props.jump(addr);
         }
 
+        console.log("Executed?");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17523-20220705-071122-4dc49800-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)